config/locales/en.yml

```diff
@@ -33,10 +33,32 @@ en:
         about_program:
           title: "About this program"
           text: "This is a program for undocumented immigrants to get financial assistance during the pandemic. The assistance is one $500 debit card per adult. Two people who live together can apply together and get $1,000. If there are more than two undocumented adults in the household, they will need to apply on their own."
+          about_accordian:
```

Comment:
`about_program` and `about_accordian` seem to be the same thing where `about_program.text` is what should be in `about_accordian.body`